### ISSUE TITLE
Scaling validation and test using the train statistics

### ISSUE THREAD

**Issue by vrodriguezf** [timeseriesAI/timeseriesAI#6]
Hi,

I was taking a look at the ROCKET notebook and I've seen that the validation and test sets are normalized using their own mean/std, instead of the one from the training set. I am not an expert on this but I remember from the fastai course that they recommended to use only the statistics from the training set to avoid bias.

In this repo, is this done like this on purpose?

Thanks!

**Comment by vrodriguezf** [comment]
Sorry I just saw that it was  `per_sample` scaling in ROCKET, I thought it was `per_feature`.

Best!
